filament/include/filament/Engine.h

```diff
@@ -380,6 +381,21 @@ class UTILS_PUBLIC Engine {
         };
         ShaderLanguage preferredShaderLanguage = ShaderLanguage::DEFAULT;
 
+        /**
+         * The action to take if the Metal backend cannot acquire a Drawable for a frame. Only
+         * affects the Metal backend.
+         *
+         * Each frame rendered requires a CAMetalDrawable texture, which is presented on-screen at
+         * the completion of each frame. These are limited and provided round-robin style by the
+         * system.
+         *
+         * - PANIC: Terminates the application and reports an error message (default).
+         * - ABORT_FRAME: Aborts execution of the current frame. The Metal backend will attempt to
+         * acquire a new drawable at the next frame.
+         */
+        MetalDrawableFailureBehavior metalDrawableFailureBehavior =
+                MetalDrawableFailureBehavior::PANIC;
```

Comment:
I would argue that this should be a setter in `PlatformMetal`. The app should be able to cast `Platform*` to `PlatformMetal*` when the backend is `Backend::METAL`. I know we already put some backend specific things in DriverConfig; so maybe we can do that here, but we should clean this up in the future.